### ISSUE TITLE
AWS Serverless Application Repository

### ISSUE THREAD

**Issue by bootstraponline** [blackboard/lambda-selenium#27]
It might be nice to publish lambda-selenium to the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)

**Comment by tsu-denim** [comment]
Yes indeed, I'm hoping to have something on there in the near future that will support running suites of jasmine/protractor tests.

**Comment by bootstraponline** [comment]
What's the current status of the lambda selenium project? It looks like development in the repo has paused.

**Comment by bootstraponline** [comment]
Awesome. Is that not a blackboard project anymore?
